controllers/operands/ssp.go

```diff
@@ -167,6 +167,8 @@ func NewSSP(hc *hcov1beta1.HyperConverged, _ ...string) (*sspv1beta1.SSP, []hcov
 		spec.NodeLabeller.Placement = hc.Spec.Workloads.NodePlacement.DeepCopy()
 	}
 
+	spec.TLSSecurityProfile = hcoutil.GetClusterInfo().GetTLSSecurityProfile(hc.Spec.TLSSecurityProfile)
```

Comment:
Maybe we can move it up to the `spec := ` section? If there is no condition here, and we should do it anyway, Maybe we can move it up to the `spec := ` section?

Reply:
done, thanks